packages/example/components/celo-logo.tsx

```diff
@@ -0,0 +1,25 @@
+export default function CeloLogo() {
+  return (
+    <svg
+      data-name="Celo Rings"
+      viewBox="0 0 950 950"
+      className="inline h-5 w-5"
+    >
+      <path
+        data-name="Top Ring"
+        d="M575 650c151.88 0 275-123.12 275-275S726.88 100 575 100 300 223.12 300 375s123.12 275 275 275zm0 100c-207.1 0-375-167.9-375-375S367.9 0 575 0s375 167.9 375 375-167.9 375-375 375z"
+        fill="#35d07f"
+      />
+      <path
+        data-name="Bottom Ring"
+        d="M375 850c151.88 0 275-123.12 275-275S526.88 300 375 300 100 423.12 100 575s123.12 275 275 275zm0 100C167.9 950 0 782.1 0 575s167.9-375 375-375 375 167.9 375 375-167.9 375-375 375z"
+        fill="#fbcc5c"
+      />
+      <path
+        data-name="Rings Overlap"
```

Comment:
this is something that is often missed but there are dark and light versions of celo logo https://celo.org/experience/brand/logo#overview

Reply:
Hmm, how does that affect the example app though? It looks like the toggle is only for the modal. Nothing happens for me at least when I toggle dark mode. 
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/21973269/172922452-67a49313-4ecc-4fca-8558-627540bdd01d.png">

Reply:
Oh i meant the version of logo that was added was the one for dark backgrounds.

Reply:
@aaronmgdr Fixed! And also it's ready for when we do support dark mode.